src/js/components/login/login.js

```diff
@@ -35,6 +35,11 @@ class LoginPage extends MaharaBaseComponent {
     backButton = (e) => {
         Router.navigate(PAGE_URL.SERVER);
     }
+    handlePasswordSubmit = (e) => {
+        if(e.charCode == 13) {
+            this.nextButton();
```

Comment:
I think the correct way to do it, would be to surround this with a &lt;form> tag, and make its onsubmit function be this.nextButton.

https://facebook.github.io/react/docs/forms.html